### ISSUE TITLE
Presence of the Force on LS Jabba's Palace Doesn't Add to LS Force Drain

### ISSUE THREAD

**Issue by BlackbeardBen** [PlayersCommittee/gemp-swccg-public#332]
I played a game yesterday were Presence of the Force was on the LS Jabba's Palace site.  It added force icons correctly, but when I force drained there (and also when clicking on the site to see the details), it did a LS force drain of the DS of only 1, instead of 2.  We double checked all the cards on table and didn't see any that might affect the results.  I didn't get a chance to check any other locations to see if they were affected.

Replay link:
https://gemp.starwarsccg.org/gemp-swccg/game.html?replayId=BenKenobi$afujqz63p69m7mim

**Comment by ckroutil** [comment]
Yes, for some reason,  at beginning of Light Side Turn #4 LS starts with 14 force icons. Then,  when it goes to LS deploy phase (turn 4), LS force icons change to 13 for no apparent reason.

**Comment by billbisco** [comment]
Same lost-modifier / AOBS class as #911.

No separate card PR. Tracked there.
